examples/cloudcontrol/resource.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: cloudcontrol.aws.upbound.io/v1beta1
+kind: Resource
+metadata:
+  annotations:
+    meta.upbound.io/example-id: cloudcontrol/v1beta1/resource
+  labels:
+    testing.upbound.io/example-name: example
+  name: example
+spec:
+  forProvider:
+    desiredState: |-
```

Comment:
nit: It should, in theory, be possible to remove the Terraform `jsonencode` operator here, this should also be okay.

Reply:
I'he tried to use just string, like this:
        {
          "ClusterName": "example",
          "Tags": [
            {
              "Key": "CostCenter",
              "Value": "IT"
            }
          ]
        }
        
 but I got the error:
 cannot get terraform setup: failed to retrieve aws credentials from aws config: request canceled, context canceled